setup.cfg

```diff
@@ -9,6 +9,9 @@ ignore_errors = True
 [mypy-applications.catmaid.tests.*]
 ignore_errors = True
 
+[mypy-create_configuration]
+ignore_errors = True
```

Comment:
So that mypy can be run without the configuration being created and CATMAID being set up fully.